sdk/cosmosdb/cosmos/src/CosmosClient.ts

```diff
@@ -21,17 +21,17 @@ import { getEmptyCosmosDiagnostics, withDiagnostics } from "./utils/diagnostics"
  * This client is used to configure and execute requests in the Azure Cosmos DB database service.
  * @example Instantiate a client and create a new database
  * ```typescript
- * const client = new CosmosClient({endpoint: "<URL HERE>", auth: {masterKey: "<KEY HERE>"}});
+ * const client = new CosmosClient({endpoint: "<URL HERE>", key: "<KEY HERE>"});
  * await client.databases.create({id: "<datbase name here>"});
  * ```
  * @example Instantiate a client with custom Connection Policy
```

Comment:
We should add some examples of instantiating a client using managed identity (essentially cover ways of connecting to CosmosDB account without using key/connection string).

Reply:
added an example of instantiating client using managed identity

Reply:
is not the aad.ts sample provides that?

Reply:
Both the examples are there in Aad.ts sample. Don't we need these sample examples for learn documentation? https://learn.microsoft.com/en-us/javascript/api/%40azure/cosmos/cosmosclient?view=azure-node-latest

Reply:
No, we can do that later!